runtime/src/bank_forks.rs

```diff
@@ -285,6 +287,7 @@ impl BankForks {
             total_squash_cache_ms += squash_timing.squash_cache_ms as i64;
         }
         let new_tx_count = root_bank.transaction_count();
+        let accounts_data_len = root_bank.load_accounts_data_len() as i64;
```

Comment:
this is the size of the accounts data modified in a block right, not the entire size of AccountsDb? Maybe should be named `accounts_data_delta_len`

Reply:
It's the entire size, but only on this fork. So more than the delta, but not the _entire_ AccountsDb.

Reply:
got it, thanks!